plugin/advisor/pg/advisor_column_required.go

```diff
@@ -25,7 +25,7 @@ type ColumnRequirementAdvisor struct {
 }
 
 // Check checks for the column requirement.
-func (adv *ColumnRequirementAdvisor) Check(ctx advisor.Context, statement string) ([]advisor.Advice, error) {
+func (_ *ColumnRequirementAdvisor) Check(ctx advisor.Context, statement string) ([]advisor.Advice, error) {
```

Comment:
I don't know why linter isn't happy about the file this PR does not touch, but 🤷‍♂️